debian/control

```diff
@@ -379,6 +379,19 @@ Description: architecture for fetching the origin of a file software heritage ar
  .
  This package contains the softwareHeritage agent programs and their resources.
 
+Package: fossology-scancode
+Architecture: any
+Depends: fossology-common, fossology-ununpack, fossology-wgetagent,
+ ${misc:Depends}
+Description: architecture for fetching license, copyright information from scancode archive.
```

Comment:
You might not wanna call it "scancode archive" and keeping the line length <= 80 is recommended.

```suggestion
Description: architecture to fetch license, copyright information from scancode
```